dracut.html

```diff
@@ -1,4202 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I like @Nowa-Ammerlaan 's suggestion here - https://github.com/dracut-ng/dracut-ng/pull/454#pullrequestreview-2156383133

Reply:
OK this would just be something for packagers to note?  The latest change does remove the .html to avoid confusion over what's the current documentation

Reply:
Ok, per https://github.com/dracut-ng/dracut-ng/pull/454#issuecomment-2416778507 I've rebased this, retaining the removal of the `dracut.html` file